src/pages/CreatePage/SpaceDetails.jsx

```diff
@@ -45,7 +49,7 @@ const SpaceDetails = ({ metrics, players }) => {
 
     // Replace with actual API call
 
-    fetch('https://localhost:7242/api/Space/CreateSpace', {
+    fetch('http://localhost:8031/api/Space/CreateSpace', {
```

Comment:
Move this API call to the api class - F1

Reply:
Will be fixed in other PR